server/router.js

```diff
@@ -123,24 +106,3 @@ ServerRouter.prototype.getHeadersForRoute = function(definition) {
   }
   return headers;
 };
-
-/**
- * Return the route definition based on a URL, according to the routes file.
- * This should match the way Express matches routes on the server, and our
- * ClientRouter matches routes on the client.
- */
-ServerRouter.prototype.match = function(pathToMatch) {
```

Comment:
It's no longer in use - I couldn't find any references to it in the code outside of tests.  Everything is working locally on test sites and change.org's main site with it too.